exporter/sentryexporter/interfaces.go

```diff
@@ -40,3 +42,21 @@ type SentrySpan struct {
 func (s SentrySpan) IsRootSpan() bool {
 	return s.ParentSpanID == ""
 }
+
+// TraceContext describes the context of the trace.
+type TraceContext struct {
+	TraceID     string `json:"trace_id"`
+	SpanID      string `json:"span_id"`
+	Op          string `json:"op,omitempty"`
+	Description string `json:"description,omitempty"`
+}
+
+// SentryTransaction describes a Sentry Transaction.
+// TODO: generate extra fields when creating envelope EventID, Type, User, Platform, SDK
+type SentryTransaction struct {
+	*sentry.Event
+	StartTimestamp time.Time    `json:"start_timestamp,omitempty"`
+	Timestamp      time.Time    `json:"timestamp"`
```

Comment:
Somehow this didn't go on (files changed since I had a pending review). Would it make sense to do this:

```suggestion
	EndTimestamp      time.Time    `json:"timestamp"`
```

Fine to serialize as `timestamp` which is a reuse of sentry event but at least the member could be more descriptive